packages/interactor/src/focused.ts

```diff
@@ -1,6 +1,16 @@
+import { perform } from './perform';
+
 /**
  * Helper function for focused filters, returns whether the given element is focused.
  */
 export function focused(element: Element): boolean {
   return element.ownerDocument.activeElement === element;
 }
+
+export const focus = perform(<E extends HTMLElement>(element: E) => {
+    element.focus();
+});
+
+export const blur = perform(<E extends HTMLElement>(element: E) => {
+    element.blur();
+});
```

Comment:
Since these become public API, maybe we could add some documentation for them?

Reply:
👍